modules/repository/repo.go

```diff
@@ -352,7 +352,11 @@ func SyncReleasesWithTags(ctx context.Context, repo *repo_model.Repository, gitR
 		}
 
 		if err := PushUpdateAddTag(ctx, repo, gitRepo, tagName, sha1, refname); err != nil {
-			return fmt.Errorf("unable to PushUpdateAddTag: %q to Repo[%d:%s/%s]: %w", tagName, repo.ID, repo.OwnerName, repo.Name, err)
+			// ref https://github.com/torvalds/linux/tree/v2.6.11
```

Comment:
```suggestion
			// ref https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tag/?h=v2.6.11
```

your link is 404, this one works

Reply:
I would prefer to do this: https://github.com/go-gitea/gitea/pull/29295#pullrequestreview-1893511989 🤣

Reply:
I would prefer a solution without a hardcoded hash, but if it has to stay, the comment link should work 😆.